distributed-job-manager/src/test/kotlin/ru/fix/distributed/job/manager/model/JobDescriptorKtTest.kt

```diff
@@ -0,0 +1,103 @@
+package ru.fix.distributed.job.manager.model
+
+import io.kotest.matchers.shouldBe
+import org.junit.jupiter.api.Test
+import ru.fix.distributed.job.manager.*
+import ru.fix.distributed.job.manager.annotation.DistributedJobId
+import ru.fix.dynamic.property.api.DynamicProperty
+import ru.fix.stdlib.concurrency.threads.Schedule
+
+internal class JobDescriptorKtTest {
+
+    @Test
+    fun `if id not provided using jobId overriding, use annotation's value of job id`() {
+        resolveJobId(TestJob()).id shouldBe "test.job"
+    }
+
+    @Test
+    fun `if id not provided, use javaClass's name for job id`() {
+        resolveJobId(TestJobWithoutJobId()).id shouldBe
+                "ru-fix-distributed-job-manager-model-JobDescriptorKtTest_TestJobWithoutJobId"
+    }
```

Comment:
Since dot is now allowed, maybe we should no longer replace dots in FCN with `-`?

Reply:
fixed